src/main/java/org/checkerframework/checker/dependencyinjection/DependencyInjectionAnnotatedTypeFactory.java

```diff
@@ -1,18 +1,164 @@
+    ProcessingEnvironment processingEnv = this.getProcessingEnv();
+    this.bindMethods.add(
+        TreeUtils.getMethod(
+            "com.google.inject.AbstractModule", "bind", processingEnv, "com.google.inject.Key<T>"));
+    this.bindMethods.add(
+        TreeUtils.getMethod(
+            "com.google.inject.AbstractModule",
+            "bind",
+            processingEnv,
+            "com.google.inject.TypeLiteral<T>"));
+    this.bindMethods.add(
+        TreeUtils.getMethod(
+            "com.google.inject.AbstractModule", "bind", processingEnv, "java.lang.Class<T>"));
+
+    this.toMethods.add(
+        TreeUtils.getMethod(
+            "com.google.inject.binder.LinkedBindingBuilder",
+            "to",
+            processingEnv,
+            "java.lang.Class<? extends T>"));
+    this.toMethods.add(
+        TreeUtils.getMethod(
+            "com.google.inject.binder.LinkedBindingBuilder",
+            "to",
+            processingEnv,
+            "com.google.inject.TypeLiteral<? extends T>"));
+    this.toMethods.add(
+        TreeUtils.getMethod(
+            "com.google.inject.binder.LinkedBindingBuilder",
+            "to",
+            processingEnv,
+            "com.google.inject.Key<? extends T>"));
     super.postInit();
   }
 
+  /* Returns true iff the argument is an invocation of AbstractModule.bind.
+   *
+   * @param methodTree the method invocation tree
+   * @return true iff the argument is an invocation of AbstractModule.bind()
+   */
+  private boolean isBindMethod(Tree methodTree) {
+    return TreeUtils.isMethodInvocation(methodTree, this.bindMethods, this.getProcessingEnv());
+  }
+
+  /* Returns true iff the argument is an invocation of AbstractModule.to() */
+  private boolean isToMethod(Tree methodTree) {
+    return TreeUtils.isMethodInvocation(methodTree, this.toMethods, this.getProcessingEnv());
+  }
+
   @Override
   protected void postAnalyze(ControlFlowGraph cfg) {
+
+    System.out.println("--------------------");
+
+    Set<Block> visited = new HashSet<>();
+    Deque<Block> worklist = new ArrayDeque<>();
+    HashMap<Integer, Integer> knownBindings = new HashMap<>();
+
+    Block entry = cfg.getEntryBlock();
+    worklist.add(entry);
+    visited.add(entry);
+
+    while (!worklist.isEmpty()) {
+      Block current = worklist.remove();
+
+      current
+          .getNodes()
+          .forEach(
+              node -> {
+                if (node instanceof MethodInvocationNode) {
+                  MethodInvocationNode methodInvocationNode = (MethodInvocationNode) node;
+
+                  System.out.printf("Method Invocation: %s\n", methodInvocationNode);
+                  System.out.printf("Operands: %s\n\n", methodInvocationNode.getOperands());
+
+                  if (methodInvocationNode.getOperands().size() >= 2) {
+                    MethodAccessNode methodAccessNode = methodInvocationNode.getTarget();
+                    Node fieldAccessNode = methodInvocationNode.getArgument(0);
```

Comment:
This name seems misleading - I think this is either the receiver parameter or the first argument to the method.